src/main/java/org/folio/rest/ResourcePaths.java

```diff
@@ -12,7 +12,7 @@ private ResourcePaths() {
   public static final String ACQUISITION_METHOD_ENDPOINT = "/orders/acquisition-methods";
   public static final String GET_ORGANIZATION_ENDPOINT = "/organizations-storage/organizations";
   public static final String ACQUISITION_UNIT_ENDPOINT = "/acquisitions-units/units";
-  public static final String LOCATIONS_ENDPOINT = "/locations";
+  public static final String LOCATIONS_ENDPOINT = "/search/consortium/locations";
```

Comment:
I suppose this endpoint will not work in non-ECS env, so we need to switch between new and one endpoints

Reply:
This is query that can help to determine do we need to use /search/consortium/locations endpoint
/orders-storage/settings?limit=1&query=key=ALLOW_ORDERING_WITH_AFFILIATED_LOCATIONS

Reply:
I tested and the approach works for snapshot environment as well. Can you please double check?

Reply:
yes, it is working on non-ecs as well, good to know, thanks
![image](https://github.com/user-attachments/assets/38ddbf1b-5123-4b62-886d-f19f72f2f3d6)

Reply:
we will use default limit 10000 after this story completion https://folio-org.atlassian.net/browse/MSEARCH-812